client/src/pages/dashboard/student/student_dash.js

```diff
@@ -31,7 +31,7 @@ const columns: GridColDef = [
     headerClassName: "section-table-header",
     hideable: false,
     valueFormatter: (v) => {
-      return v.value + " QL";
+      return v?.value !== null ? v.value + " QL" : "";
```

Comment:
Personally, for nullish, I prefer `!= null`, which will also include undefined.
```suggestion
      return v?.value != null ? v.value + " QL" : "";
```